packages/cyberstorm/package.json

```diff
@@ -18,6 +18,8 @@
     "@fortawesome/fontawesome-svg-core": "^6.2.0",
     "@fortawesome/free-solid-svg-icons": "^6.2.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@radix-ui/react-dropdown-menu": "^2.0.1",
+    "@radix-ui/react-select": "^1.1.2",
```

Comment:
Running `yarn.install` seems to alter `yarn.lock` on my machine. Did you remember to update `yarn.lock` in this PR after react-select was added?

Reply:
Should be updated in the next commit